tools/fonts/@/gonvillesmufl/LICENSE.txt

```diff
@@ -0,0 +1,83 @@
+Copyright (c) 2022, Rodrigo J. Vilar de Linares (rvilarl@gmail.com),
```

Comment:
Can we add a README.txt in this same folder, or add a short notice in this LICENSE file that GonvilleSmufl is a SMuFL compatible derivative of Gonville:

https://www.chiark.greenend.org.uk/~sgtatham/gonville/

I know the original website says:

"The Gonville font files themselves may be used without restriction. You don't even have to preserve a copyright notice or anything. Use freely and enjoy."

But it's still nice to add attribution to the original font designer:  Simon Tatham

Reply:
I added the readme from github.com/rvilarl/gonvillesmufl. I was planning to move the repository to the vexflow organization that you proposed but this never happened.